src/runtime/virtcontainers/mount.go

```diff
@@ -30,6 +30,11 @@ const UmountNoFollow = 0x8
 
 var rootfsDir = "rootfs"
 
+var lowerDir = "lowerdir"
```

Comment:
Should these be const? and `fs`/`work` below.

Reply:
done